system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs

```diff
@@ -601,6 +611,9 @@ pub type XcmRouter = WithUniqueTopic<(
 	// Router which wraps and sends xcm to BridgeHub to be delivered to the Polkadot
 	// GlobalConsensus
 	ToPolkadotXcmRouter,
+	// Router which wraps and sends xcm to BridgeHub to be delivered to the Ethereum
+	// GlobalConsensus
+	SovereignPaidRemoteExporter<bridging::EthereumNetworkExportTable, XcmpQueue, UniversalLocation>,
```

Comment:
```suggestion
	SovereignPaidRemoteExporter<
		xcm_builder::NetworkExportTable<to_ethereum::BridgeTable>,
		XcmpQueue,
		UniversalLocation
	>,
```

My original idea for extending more routings was to simply use `BridgeTable` and add a new `.chain()` function. However, I now realize I was mistaken (as it could potentially introduce a bug where we allow routing with the wrong router). I will prepare another PR to refactor `pub BridgeTable:`.

So, if you don't need `bridging::EthereumNetworkExportTable` elsewhere, I suggest removing it along with `pub EthereumBridgeTable` as they are not really needed.

Reply:
Removed in https://github.com/polkadot-fellows/runtimes/pull/130/commits/eed8cd9341914cb869328ea79d79cc9e52d27daf and https://github.com/polkadot-fellows/runtimes/pull/130/commits/3bc7a566730eff282f1a74ac07d370e12c0fe9e0.